docs/data-sources/source.md

```diff
@@ -17,39 +17,48 @@ This Data Source allows you to look up existing Logtail Sources using their tabl
 
 ### Required
 
-- **table_name** (String) The table name generated for this source.
+- `table_name` (String) The table name generated for this source.
```

Comment:
Are these formatting changes intentional?
This is the "standard" format for Terraform docs AFAIK, as generated by https://github.com/BetterStackHQ/terraform-provider-logtail/blob/main/Makefile#L69

Reply:
I ran `tfplugindocs generate` and this is what came out, not sure if it's supposed to look like this, I assumed that the source uses `**` instead of backticks since Go uses backticks for string literals

Reply:
Looks like `tfplugindocs` changed this at some point, old version used stars, new uses backticks